test/unit/connection_string.spec.test.ts

```diff
@@ -1,27 +1,20 @@
-import { expect } from 'chai';
-
-import { parseOptions } from '../../src/connection_string';
 import { loadSpecTests } from '../spec';
+import { executeUriValidationTest } from '../tools/uri_spec_runner';
 
-// NOTE: These are cases we could never check for unless we write our own
-//       url parser. The node parser simply won't let these through, so we
-//       are safe skipping them.
 const skipTests = [
-  'Invalid port (negative number) with hostname',
-  'Invalid port (non-numeric string) with hostname',
+  // TODO: supposedly this one test should be caught by node, but it isn't
```

Comment:
This is because our connection string is not a standard URI and can contain multiple hosts/port pairs in it. We defined the trailing "/" as required to specify that the host information has ended in the string.

Reply:
@durran Thanks for the context! (@nbbeeken fyi) - seems like we should add this validation in our connection string module then

Reply:
Filed NODE-3919

Reply:
That's interesting I don't see why it would be needed, single host or comma separated host, a question mark is needed to being the searchParams section of a URI, so I'm not sure the scenario where you _need_ the slash to indicate the host information is complete, but we can certainly take a closer look in a follow up!

Reply:
I'd ask Ross for the exact reasoning as I wasn't around in drivers at the time, but it is in the spec: https://github.com/mongodb/specifications/blob/master/source/connection-string/connection-string-spec.rst#host-information